akka-docs/_sphinx/themes/akka/layout.html

```diff
@@ -23,7 +23,9 @@
 {% block relbar2 %}{% endblock %}
 
 {% block extrahead %}
-  <!-- Akka release version-->
+    <!-- Hint to search engines that the "canonical" page is under "current", which will boost it appearing in search results -->
+    <link rel="canonical" href="http://akka.io/docs/akka/current/{{ pagename }}.html" />
```

Comment:
This should probably only be in the online version (i.e. in the `if include_analytics` block).